index.html

```diff
@@ -71,6 +72,110 @@ <h2>Software Developer</h2>
         </div>
 
     </footer>
+    <section class="work-sect">
+        <div>
+            <h2>MY RECENT WORK</h2>
+        </div>
+        <div class="work-sect">
+            <img class="card-image" src="/assets/images/dummy-image-24toolbox-350x450.png" alt="dummy image">
+            <h3>Multi-Post Stories</h3>
+            <p>
+                A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+                industry's
+                standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+            </p>
+
+            <ul class="grid-container">
+                <li>Css</li>
+                <li class="border1">Html</li>
+                <li class="border2">Bootstrap</li>
+                <li>Ruby</li>
+            </ul>
+            <button class="btn">See Project</button>
+
+
+
+        </div>
+
+    </section>
+    <section class="work-sect">
+        <div>
+            <h2>MY RECENT WORK</h2>
+        </div>
+        <div class="work-sect">
+            <img class="card-image" src="/assets/images/dummy-image-24toolbox-350x450.png" alt="dummy image">
+            <h3>Multi-Post Stories</h3>
```

Comment:
- [ ] My Recent Work is the section title. You have repeated it for every project. You should only have one for the entire section